tests/unittests/thermal_heuristic/test_data.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) 2024, RTE (https://www.rte-france.com)
+#
+# See AUTHORS.txt
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of the Antares project.
+
+import numpy as np
+import pandas as pd
+
+from andromede.thermal_heuristic.data import (
+    get_max_failures,
+    get_max_unit,
+    get_max_unit_for_min_down_time,
+    shift_df,
+)
+
+
+def test_get_max_unit() -> None:
+    max_generating = pd.DataFrame([0, 10, 10, 15, 20, 5, 25])
+
+    max_unit = get_max_unit(pmax=10, units=2, max_generating=max_generating)
```

Comment:
Could we get rid of this function if we have appropriate operators in expressions such as floor, ceil, max ?